Specs/Sequence/English/PhoneNumberModel.json

```diff
@@ -883,6 +883,2692 @@
+        "Resolution": {
+          "value": "+44 161 123 4567"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "My phone number is +441611234567.",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "+441611234567",
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "value": "+441611234567"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "My phone number is +44(0)161234567.",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "+44(0)161234567",
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "value": "+44(0)161234567"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "My phone number is 00 44 161 1234567.",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "00 44 161 1234567",
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "value": "00 44 161 1234567"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "My phone number is (011) 44 161 234567.",
+    "NotSupportedByDesign": "python",
+    "NotSupported": "dotnet,javascript",
+    "Results": [
+      {
+        "Text": "(011) 44 161 234567",
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "value": "(011) 44 161 234567"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "My phone number is 0161-158-5587.",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "0161-158-5587",
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "value": "0161-158-5587"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "My phone number is 0161 123 4567 ext. 123.",
+    "NotSupportedByDesign": "python",
+    "NotSupported": "dotnet,javascript",
+    "Results": [
+      {
+        "Text": "0161 123 4567 ext. 123",
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "value": "0161 123 4567 ext. 123"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "My phone number is 01611234567x123.",
+    "NotSupportedByDesign": "python",
+    "NotSupported": "dotnet,javascript",
+    "Results": [
+      {
+        "Text": "01611234567x123",
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "value": "01611234567x123"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "My phone number is +44161234567x123.",
+    "NotSupportedByDesign": "python",
+    "NotSupported": "dotnet,javascript",
+    "Results": [
+      {
+        "Text": "+44161234567x123",
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "value": "+44161234567x123"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "My phone number is +44 (0) 161 1234567 ext 123.",
+    "NotSupportedByDesign": "python",
+    "NotSupported": "dotnet,javascript",
+    "Results": [
+      {
+        "Text": "+44 (0) 161 1234567 ext 123",
+        "TypeName": "phonenumber",
+        "Resolution": {
+          "value": "+44 (0) 161 1234567 ext 123"
+        }
```

Comment:
do we need to include the invalid exmaples in the website which you share?

Reply:
and you also can add "911", "110"

Reply:
Not at this point, due to conflicts.

Reply:
but can mark them as "notsupported"